Trio/Sources/Modules/Home/View/HomeRootView.swift

```diff
@@ -467,31 +496,38 @@ extension Home {
                         .font(.callout)
                 } else {
                     HStack {
-                        if state.pumpSuspended {
+                        if state.apsManager?.isScheduledBasal == nil {
+                            /// The pump is not currently available (e.g., no pod) so
+                            /// display no insulin delivery info rather than "Pump suspended"
+                        } else if state.apsManager.isSuspended {
                             Text("Pump suspended")
                                 .font(.callout).fontWeight(.bold).fontDesign(.rounded)
                                 .foregroundColor(.loopGray)
-                        } else if let tempBasalString = tempBasalString {
+                        } else {
                             Image(systemName: "drop.circle")
                                 .font(.callout)
                                 .foregroundColor(.insulinTintColor)
-                            if tempBasalString.count > 5 {
-                                Text(tempBasalString)
-                                    .font(.callout).fontWeight(.bold).fontDesign(.rounded)
-                                    .lineLimit(1)
-                                    .minimumScaleFactor(0.85)
-                                    .truncationMode(.tail)
-                                    .allowsTightening(true)
+                            if let basalString = self.basalString {
+                                // If running a scheduled basal, display basalString in loopGray instead of black
+                                let color: Color = state.apsManager?.isScheduledBasal == true ? .loopGray : .black
```

Comment:
The color can stay the same if it's the correct value, no need to change it.

We will very likely remove this information entirely in upcoming home screen changes to enable better accessibility of Trio.